apps/gitness/src/pages/pull-request/pull-request-conversation-page.tsx

```diff
@@ -52,6 +52,7 @@ export default function PullRequestConversationPage() {
   } = usePullRequestData()
   const { currentUser: currentUserData } = useAppContext()
   const [checkboxBypass, setCheckboxBypass] = useState(false)
+  const { spaceId, repoId } = useParams<PathParams>()
```

Comment:
maybe use the custom hooks we have in the framework for this?

Reply:
didn't use  useGetRepoRef since its both repoid and spaceId

and it seemed shorter to just export both from useParams